app/models/Document.ts

```diff
@@ -105,8 +105,6 @@ export default class Document extends ArchivableModel implements Searchable {
         return "Markdown";
       case FileOperationFormat.JSON:
         return "JSON";
-      case FileOperationFormat.Notion:
```

Comment:
Retain this reference to `FileOperationFormat.Notion` enum for backwards compat with existing imports